eng/coredistools.targets

```diff
@@ -0,0 +1,15 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
```

Comment:
Should the whole file import instead be conditioned as the properties below won't work without the PackageReference.

Reply:
I can move this condition to `Import` element in R2RDump.csproj. 

Would I need the same condition for `src/tests/Common/Directory.Build.targets` ? As far as I understand, we don't build tests during source build, so the condition will always be true.

Reply:
Yeah please add it to both imports. You are right that we don't build these currently in source build we might at some point.

Reply:
Thanks @ViktorHofer - I addressed both of your suggestions.